packages/docz-plugin-vue/src/index.ts

```diff
@@ -0,0 +1,39 @@
+import { createPlugin } from 'docz-core'
+import { VueLoaderPlugin } from 'vue-loader'
+
+interface BabelRC {
+  presets?: any[]
+  plugins?: any[]
+  cacheDirectory?: boolean
+  babelrc?: boolean
+}
+
+export const doczPluginVue = () =>
+  createPlugin({
+    modifyBabelRc: (babelrc: BabelRC): BabelRC => {
+      if (babelrc.plugins === undefined) {
+        babelrc.plugins = []
+      }
+      babelrc.plugins.push('vuera/babel')
+      babelrc.babelrc = true
+      return babelrc
+    },
+
+    modifyBundlerConfig: config => {
+      config.module.rules.push({
+        test: /\.vue$/,
+        use: 'vue-loader',
+        exclude: /node_modules/,
+      })
+
+      config.module.rules.push({
+        test: /\.css$/,
```

Comment:
uma parada que tava pensando, é como integrar isso com o [docz-plugin-css](https://github.com/pedronauck/docz/tree/master/packages/docz-plugin-css) 🤔

Reply:
That should be a good thing too, we could specify it in Playground tag or also in configuration file, in vue we can just set a tag `lang="scss"` to change the processor of `template`, `style` and `script`. Might be a way to get it. https://vue-loader.vuejs.org/guide/pre-processors.html#sass